README.md

```diff
@@ -7,7 +7,7 @@ This is a [homebrew tap](https://docs.brew.sh/Taps) for [nostr](https://nostr.co
 First install [homebrew](https://brew.sh/). Then add the tap:
 
 ```
-brew tap nostorg/nostr
+brew tap nostorg/homebrew-nostr
```

Comment:
This is not necessary, taps on github under `user/homebrew-repo` are tapped as `brew tap user/repo`.

Reply:
I stated as such in the commit message...

![Screen Shot 2023-10-01 at 10 38 19 AM](https://github.com/nostorg/homebrew-nostr/assets/152159/73bb4572-8b89-4dfc-9cff-f62ad974a595)

Reply:
It is better to use the actual syntax versus the "alias".

Reply:
As you can see - it lands in the Taps directory as homebrew-nostr
so why not refer to it as it actually exists? 

![Screen Shot 2023-10-01 at 10 48 02 AM](https://github.com/nostorg/homebrew-nostr/assets/152159/541d9622-f807-4828-b3df-bf137011f1ad)

Reply:
Because that is NOT the correct syntax. https://docs.brew.sh/Taps

Even if you use the fully qualified 2-argument form of `brew tap` with `user/homebrew-repo` it still recognizes it internally as `user/repo`:

```
❯ brew tap nostorg/homebrew-nostr https://github.com/nostorg/homebrew-nostr
==> Tapping nostorg/nostr
Cloning into '/home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/nostorg/homebrew-nostr'...
remote: Enumerating objects: 300, done.
remote: Counting objects: 100% (86/86), done.
remote: Compressing objects: 100% (55/55), done.
remote: Total 300 (delta 42), reused 67 (delta 31), pack-reused 214
Receiving objects: 100% (300/300), 40.83 KiB | 929.00 KiB/s, done.
Resolving deltas: 100% (116/116), done.
Tapped 5 casks and 7 formulae (26 files, 141.8KB).
❯ brew tap
nostorg/nostr
```